worker/src/main/java/org/apache/celeborn/service/deploy/worker/memory/MemoryManager.java

```diff
@@ -65,8 +67,11 @@ public class MemoryManager {
   private final AtomicLong diskBufferCounter = new AtomicLong(0);
   private final LongAdder pausePushDataCounter = new LongAdder();
   private final LongAdder pausePushDataAndReplicateCounter = new LongAdder();
+  private final LongAdder pausePushDataTime = new LongAdder();
```

Comment:
Since `switchServingState` will be called sequentially in a single thread, I think it's safe to use long/int for `pausePushDataTime` and `trimCounter`